src/datanode/src/datanode.rs

```diff
@@ -148,20 +148,26 @@ impl From<&DatanodeOptions> for StorageEngineConfig {
 #[serde(default)]
 pub struct ProcedureConfig {
     /// Storage config for procedure manager.
-    pub store: ObjectStoreConfig,
+    pub store_path: String,
```

Comment:
issue: We might want to store procedure on object store services. I prefer to keep the original field unchanged.
```suggestion
    pub store_path: ObjectStoreConfig,
```